Samples/ReplicatedChatGrainSample/QueuedGrainSample.Grains/ChatGrain.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ReplicatedChatGrainSample.Interfaces;
+using Orleans;
+using Orleans.Providers;
+using Orleans.Concurrency;
+using Orleans.EventSourcing;
+
+namespace ReplicatedChatGrainSample.Grains
+{
+    [Serializable]
+    public class ChatState
```

Comment:
Does this class even need to be serializable? Because I tried locally by removing this attribute and it worked. If it doesn't need to be, I can remove the attribute, as it would be less confusing... I wasn't expecting this to have to be serializable, unless the provider implements snapshotting or something like that.
I'm in the process of doing more cleaning, so I can remove it myself if you say this isn't needed